packs/st2/aliases/actions_list.yaml

```diff
@@ -1,4 +1,7 @@
 ---
-name: "st2-actions-list"
+name: "st2_actions_list"
 action_ref: "st2.actions.list"
 description: "List available StackStorm actions."
+formats:
+    - "list actions"
```

Comment:
I think we should probably still prefix all those commands with "st2" (e.g. `!st2 list actions`). 

What do other ppl think?